### DISCUSSION TITLE
Helm chart template issue

### DISCUSSION THREAD

**Discussion by iWangJiaxiang** [requarks/wiki#3602] [Error / Bug Report]
**Describe the bug**

Hi, I'm experiencing a chart template issue when I tried to deploy wiki.js in my k8s cluster. 

The error log:

```
Failed to install app wiki. Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(Service.spec.ports[1]): missing required field "port" in io.k8s.api.core.v1.ServicePort
```

I rendered the helm chart template locally and found an error like this:

![image](https://user-images.githubusercontent.com/19409802/89365964-61dff980-d708-11ea-8632-e5f671e4adf0.png)

The value `{{.Values.service.httpsPort}}`` was commented so there was nothing but empty, which caused the issue.

![image](https://user-images.githubusercontent.com/19409802/89366331-44f7f600-d709-11ea-9e1b-4de091bb62cf.png)

**To Reproduce**

1. Using current helm chart from `dev` branch
2. Set persistence values
3. Deploy chart

**Expected behavior**

The wiki.js should be deployed successfully.

**Screenshots**

![image](https://user-images.githubusercontent.com/19409802/89365716-cc446a00-d707-11ea-9e39-e65003fe3241.png)

**Host Info (please complete the following information):**
 - OS: Kubernetes
 - Wiki.js version: 2.4.107

**Additional context**

Currently the work around is setting the value `service.httpsPort` to render the correct value

**Comment by jochen-st** [comment]
I believe this was fixed with https://github.com/Requarks/wiki/pull/2472
